crates/router/src/connector/nuvei/transformers.rs

```diff
@@ -1347,6 +1371,18 @@ fn get_payment_status(response: &NuveiPaymentsResponse) -> enums::AttemptStatus
     }
 }
 
+fn get_device_details(
```

Comment:
This could be try_from rather than a method, Since this holds only type conversion logic